module-1/web-project/your-code/readme_miriam.md

```diff
@@ -5,11 +5,11 @@
 Después de investigar varias API procedentes de los dos enlaces proporcionados en la práctica. Me he decicio por "Makeup API" (http://makeup-api.herokuapp.com/). Una API que no requiere autentificación y tiene diferentes datos de 930 productos de belleza de una web. <br>
```

Comment:
Really good Readme Miriam!! You only missed to save the results of your API and scraping job in an output folder!!